subsys/bluetooth/audio/unicast_client.c

```diff
@@ -2507,7 +2514,10 @@ static uint8_t unicast_client_read_func(struct bt_conn *conn, uint8_t err,
 	while (rsp->num_pac) {
 		struct unicast_client_pac *bpac;
 		struct bt_pac *pac;
-		struct bt_pac_meta *meta;
+		struct bt_pac_ltv *meta_hdr, *cc_hdr;
```

Comment:
Here you actually use the `ltv` to store the `hdr`, so perhaps `bt_pac_ltv_hdr` would be a better name? 

It is a bit confusing with the current names where you use the `_ltv` to _not_ store the actual LTV, but rather the header _and_ the LTV.